src/wifi.cpp

```diff
@@ -16,7 +16,11 @@ IPAddress apIP(192, 168, 4, 1);
 IPAddress netMsk(255, 255, 255, 0);
 
 // hostname for mDNS. Should work at least on windows. Try http://openevse or http://openevse.local
+#ifdef HOSTNAME
```

Comment:
Personally I prefer to always have 
HOSTNAME set using;
```
#ifndef HOSTNAME
#define HOSTNAME openevse
#endif
```
somewhere near the top of the file, see [wifi.cpp](https://github.com/OpenEVSE/ESP8266_WiFi_v2.x/blob/master/src/wifi.cpp#L30)